homeassistant/components/automate/sensor.py

```diff
@@ -17,13 +17,17 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     hub = hass.data[DOMAIN][config_entry.entry_id]
 
     current_battery = set()
+    current_voltage = set()
     current_signal = set()
 
     @callback
     def async_add_automate_sensors():
         async_add_automate_entities(
             hass, AutomateBattery, config_entry, current_battery, async_add_entities
         )
+        async_add_automate_entities(
+            hass, AutomateVoltage, config_entry, current_voltage, async_add_entities
+        )
         async_add_automate_entities(
```

Comment:
Side note for the future: It's not very efficient to make multiple calls to `async_add_entities`. It would be good to refactor this so that we only need to call `async_add_entities` once per signal.